modules/administration/pages/actions.adoc

```diff
@@ -16,14 +16,22 @@ Remote commands allows you to issue commands to individual Salt clients, or to a
 
 == Recurring Actions
 
-You can apply recurring actions on individual Salt clients, or to all clients in an organization.
+You can apply recurring actions on individual Salt clients, to a system group, or to all clients in an organization.
+
+Currently, {productname} supports the following action types as recurring actions:
+
+- [parameter]``Highstate:`` Execute the highstate
+- [parameter]``Custom state:`` Execute a set of custom states. A custom state can be either an internal state provided by {productname}, or a configuration channel created by a user.
```

Comment:
```suggestion
- [parameter]``Custom state:`` Execute a set of custom states. A custom state can be either an internal state provided by {productname}, or a state channel created by a user.
```

nitpick ^^

Reply:
Actually, both state channels and normal config channels can be added as a state to recurring actions